src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/ReleaseRequests/ReleaseRequests.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { type FC } from 'react';
```

Comment:
I've added new shared component and changed `FundingRequests` and `FinalizeRequests` but left it in `ReleaseActions` because there is some additional logic there